packages/components/src/components/filter-type-group/filter-search/filter-search.tsx

```diff
@@ -0,0 +1,38 @@
+import { Component, h, Listen, Watch, Element, Event, EventEmitter, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-filter-search',
+  styleUrl: 'filter-search.scss',
+  shadow: true,
+})
+export class FilterSearch {
+  @Element() host: HTMLElement;
+  @Prop() filterName: string;
+  @Prop() disabled: boolean = false;
+  @Prop({ mutable: true }) filterValue: string;
+  @Event() ifxFilterSearchChange: EventEmitter;
+
+
+  @Watch('value')
```

Comment:
What is the purpose of this? There is no prop called `"value"`, so this `@Watch` decorator is not listening to anything.